### PR TITLE
Merge relevant changes from the sles-build branch.

### PR DESCRIPTION
**Pull request by igorpeshansky**
Upgrade omnibus to pick up https://github.com/chef/chef/blob/master/RELEASE_NOTES.md#sles-15-detection.

Using omnibus 6.1.7 because 6.1.8 introduced a bug that was only fixed after 6.1.9 (the latest) was released. The omnibus-software commit is from the same date as omnibus 6.1.7, to keep them in sync.

### CODE REVIEW

File: templates/etc/init.d/rpm/td-agent [hunk 1]
```diff
@@ -148,15 +146,15 @@ if [ -n "${TD_AGENT_USER}" ]; then
...
+  START_STOP_DAEMON_ARGS="${START_STOP_DAEMON_ARGS} -u ${TD_AGENT_USER}"
 fi
 
 if [ -n "${TD_AGENT_GROUP}" ]; then
   if ! getent group | grep -q "^${TD_AGENT_GROUP}:"; then
     echo "$0: group for running ${TD_AGENT_NAME} doesn't exist: ${TD_AGENT_GROUP}" >&2
     exit 1
   fi
-  TD_AGENT_ARGS="${TD_AGENT_ARGS} --group ${TD_AGENT_GROUP}"
+  TD_AGENT_ARGS="${TD_AGENT_ARGS} -g ${TD_AGENT_GROUP}"
```
Comment (sophieyfang) [inline templates/etc/init.d/rpm/td-agent]: -g replaces --group for simplicity? Same as above for -u v.s. --user

Comment (igorpeshansky) [reply templates/etc/init.d/rpm/td-agent]: Well, we did switch from calling `daemon` (which supports `--user` and `--group`) to calling `start_daemon` (which supports `-u` and `-g`). However, you did catch a bug (thanks): the `-g` argument was supposed to be passed to `start_daemon`, not `td-agent`. Just pushed a commit with a fix.

**Comment by igorpeshansky** [comment]
Fixed some bugs. This is now ready for another review. PTAL.

File: templates/etc/init.d/rpm/td-agent (hunk 1 above)
Comment (igorpeshansky) [reply templates/etc/init.d/rpm/td-agent]: Turns out `-g` doesn't work on CentOS/RHEL, so I've restored the original code.

**Pull request merged**